### PR TITLE
[GUI] Translation cleanup

### PR DESCRIPTION
**Pull request by Fuzzbawls**
This stops the practice of repeatedly translating the string literal `PIVX Core` in multiple strings, which should never be localized as it is a proper product name.

Also stops translating command line arguments as they were only ever translated for pivx-qt and not for the command line programs. Many of these arguments are highly technical in nature, and translating them has been problematic (with translation errors or literal translations that result in incorrect argument names) and leads to more confusion rather than less confusion.

Lastly, a quick formatting change to the `-help` output of binary programs is included to make the release-distributed manpages more compliant with the help2man expected formatting.

Overall, this results in a reduction of strings to translate by ~190, which should assist in getting language translations to meet our minimum translation threshold for inclusion in releases.

-------

Based on the following upstream commits/PRs:
- https://github.com/bitcoin/bitcoin/pull/7192/commits/d5f46832de900cee0801ca40bba743c9564cccb8
- https://github.com/bitcoin/bitcoin/pull/13341
- https://github.com/bitcoin/bitcoin/pull/13872
- https://github.com/bitcoin/bitcoin/pull/16291

### CODE REVIEW

File: doc/man/pivx-cli.1 [hunk 1]
```diff
@@ -1,19 +1,21 @@
...
-PIVX Core RPC client version v3.1.99.0\-11f4a2bba4\-dirty
-.SS "Usage:"
-.TP
-pivx\-cli [options] <command> [params]
-Send command to PIVX Core
-.TP
-pivx\-cli [options] help
-List commands
-.TP
-pivx\-cli [options] help <command>
```
Comment (Gordis890) [inline doc/man/pivx-cli.1:15]: doc/man/pivx-cli. 1

File: src/wallet/wallet.cpp [hunk 2]
```diff
@@ -4129,10 +4133,10 @@ CWallet* CWallet::CreateWalletFromFile(const std::string& name, const fs::path&
             UIWarning(strprintf(_("Warning: error reading %s! All keys read correctly, but transaction data"
                          " or address book entries might be missing or incorrect."), walletFile));
         } else if (nLoadWalletRet == DB_TOO_NEW) {
-            UIError(strprintf(_("Error loading %s: Wallet requires newer version of PIVX Core"), walletFile));
+            UIError(strprintf(_("Error loading %s: Wallet requires newer version of %s"), walletFile, PACKAGE_NAME));
             return nullptr;
         } else if (nLoadWalletRet == DB_NEED_REWRITE) {
-            UIError(_("Wallet needed to be rewritten: restart PIVX Core to complete"));
+            UIError(strprintf(_("Wallet needed to be rewritten: restart %s to complete"), PACKAGE_NAME));
             return nullptr;
```
Comment (Gordis890) [inline src/wallet/wallet.cpp:4140]: Src/approved/wallet/wallet. Ccp

File: doc/man/pivx-cli.1 (hunk 1 above)
Comment (Fuzzbawls) [reply doc/man/pivx-cli.1:15]: nonsense comment, dismissing

File: src/wallet/wallet.cpp (hunk 2 above)
Comment (Fuzzbawls) [reply src/wallet/wallet.cpp:4140]: nonsense comment, dismissing

**Review by furszy — APPROVED** [review]
ACK d838f20

**Review by random-zebra — APPROVED** [review]
utACK d838f2022972dc4f8446bfc7707c065095445507. Merging.

**Pull request merged**